content/news/033/index.md

```diff
@@ -72,6 +72,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Learning Material Updates
 
+### Game Development with Rust and WebAssembly
+
+![Game Development with Rust and WebAssembly Book Cover](https://user-images.githubusercontent.com/21752/166943991-09f449bc-27f1-431e-ad32-a3ab8ccbcbbc.png)
```

Comment:
Could you please commit the image to the repo too? Note that it should be under 300kb.

Reply:
👍